libs/utils/include_deprecated/linked_list.h

```diff
@@ -32,63 +32,57 @@
 #include "celix_errno.h"
 #include "exports.h"
 
-#ifdef ADD_CELIX_DEPRECATED_WARNING
-#define CELIX_DEPRECATED_ATTR __attribute__ ((deprecated))
-#else
-#define CELIX_DEPRECATED_ATTR
-#endif
-
 #ifdef __cplusplus
 extern "C" {
 #endif
-typedef struct linked_list_entry *linked_list_entry_pt; //TODO make deprecated
-typedef struct linked_list *linked_list_pt; //TODO make deprecated
+typedef struct linked_list_entry *linked_list_entry_pt;
```

Comment:
There was a note with `TODO make deprecated`. Why only mark the `linked_list_pt` as deprecated and not this one?

Reply:
> There was a note with `TODO make deprecated`. Why only mark the `linked_list_pt` as deprecated and not this one?

Now linked_list_entry is also marked as deprecated